standard/expressions.md

```diff
@@ -1129,6 +1129,8 @@ primary_no_array_creation_expression
     ;
 ```
 
+> *Note*: These grammar rules are not ANTLR-ready as they are part of a set of mutually left-recursive rules (`primary_expression`, `primary_no_array_creation_expression`, `member_access`, `invocation_expression`, `element_access`, `post_increment_expression`, `post_decrement_expression`, `pointer_member_access` and `pointer_element_access`) which ANTLR does not handle. Standard techniques can be used to transform the grammar to remove the mutual left-recursion. This has not been done as not all parsing strategies require it (e.g. an LALR parser would not) and doing so would obfuscate the structure and description.
```

Comment:
This is the one mutual left recursive set of rules being left. To factor the rules in the list can be inlined into `primary_no_array_creation_expression` (see the Parse_CSharp.g4 file sent out). If you've an opinion on removing/leaving the MLR please comment, or would like propose an alternative grammar please do so!

Reply:
I agree with your comment in the change that a refactoring would obscure the structure.

Reply:
I agree as well.